app/exports/common_columns/_provider_information.yml

```diff
@@ -39,7 +39,13 @@ course_level:
 
 program_type:
   type: string
-  description: Not provided yet
+  description: The program type
+  enum:
+    - scitt_programme
+    - school_direct_training_programme
+    - higher_education_programme
+    - school_direct_salaried_training_programme
+    - pg_teaching_apprenticeship
```

Comment:
I couldn't find a definitive list for these, so took my best guess. Could someone from the ProVendor team confirm these please?

Reply:
Cheers Ben! 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/34ff311899125761b3bbeac3d7735540bbf3f503/app/models/course.rb#L43

Reply:
These are defined in the enum in the course object. More generally, the values for this come from Find when we sync